web-common/src/features/explore-mappers/getDashboardFromComponentTransformer.ts

```diff
@@ -0,0 +1,21 @@
+import type { ExploreState } from "@rilldata/web-common/features/dashboards/stores/explore-state";
+import type {
+  ComponentTransformerProperties,
+  TransformerArgs,
+} from "@rilldata/web-common/features/explore-mappers/types";
+
+export function getDashboardFromComponentTransformer({
```

Comment:
It feels overkill to add this here if all it does is merge with default dashboard.

`getExplorePageUrlSearchParams` returns the url for a partial state itself. The other methods were updated to accept `Partial<ExploreState>` but this seems to be missed. So no need to call `mapObjectToExploreState`, you could pass around `Partial<ExploreState>`